examples/example_fastf1_interactive_graph.py

```diff
@@ -0,0 +1,117 @@
+"""
+Interactive graph for laptimes
+=================================
+
+Plot an interactive graph showing laptimes for each driver lap by lap.
+"""
+
+import fastf1
+import fastf1.plotting
+import plotly.graph_objects as go
+from plotly.io import show
+
+# SETTING UP THE DATA #########################################################
+# load a session
+race = fastf1.get_session(2023, 'Abu Dhabi', 'R')
+race.load(weather=False, telemetry=False)
+
+# Get the list of drivers
+driver_list = list(fastf1.plotting.DRIVER_TRANSLATE.keys())
+
+# Set the colors for the graph: for drivers, we use the combination of both
+# DRIVER_TRANSLATE and DRIVER_COLORS to associate colors with driver's initials
+driver_colors = {
+    key: fastf1.plotting.DRIVER_COLORS[value]
+    for key, value in fastf1.plotting.DRIVER_TRANSLATE.items()
+    }
+
+# Set the colors of each tyre
+compound_colors = fastf1.plotting.COMPOUND_COLORS
+
+# Set titles and labels of the graph
+title = "2023 Abu Dhabi Grand Prix Laptime Evolution"
+xaxis_title = "Lap Number"
+yaxis_title = "Lap Time (s)"
+legend_title = "Driver"
+###############################################################################
+
+
+# DEFINING THE LIMITS OF THE GRAPH#############################################
+# Define the limits of the graph:
+# As we are going to show a representation of the whole race, there are
+# some laps that are considerably slower than the average. We want to keep them
+# in the graph but we don't need to see them, so we're focusing our view in the
+# relevant ones. For this reason, we're picking the fastest lap and the
+# slowest lap of the quick laps and we're defining a margin of 0'5s for the
+# limits of the graph.
+quicklaps = race.laps.pick_quicklaps()
+quicklaps['LapTime(s)'] = quicklaps['LapTime'].dt.total_seconds()
```

Comment:
`.pick_quicklaps` returns a slice of the entire laps data. Therefore, setting `quicklaps['LapTime(s)']` sets the new column of a copy of that slice which triggers the pandas warning.
Either, use `race.laps.pick_quicklaps().copy()` or just don't set the lap time in seconds as a new column. It could just be its own variable.
Please do not suppress the warning, that's just confusing as well.